libs/sdk-backend-spi/src/backend/capabilities.ts

```diff
@@ -211,6 +211,12 @@ export interface IBackendCapabilities {
      */
     supportsSeparateLatitudeLongitudeLabels?: boolean;
 
+    /**
+     * Indicates whether backends supports displaying message to inform workspace managers about having
+     * access to the shared object.
+     */
+    canWorkspaceManagerSeeEverySharedObject?: boolean;
```

Comment:
I'm not sure about this naming :/ looks a bit strange

Reply:
We wanted to avoid the keyword `admin` and `dashboard` so this is what came out of `canWorkspaceAdminSeeEveryDashboard`